sktime/forecasting/sarimax.py

```diff
@@ -0,0 +1,182 @@
+        iterable defining the non-zero polynomial exponents to include, in
+        increasing order. For example, `[1,1,0,1]` denotes
+        :math:`a + bt + ct^3`. Default is to not include a trend component.
+    measurement_error : bool, optional, default=False
+        Whether or not to assume the endogenous observations `endog` were
+        measured with error.
+    time_varying_regression : bool, optional, default=False
+        Used when an explanatory variables, `exog`, are provided
+        to select whether or not coefficients on the exogenous regressors are
+        allowed to vary over time.
+    mle_regression : bool, optional, default=True
+        Whether or not to use estimate the regression coefficients for the
+        exogenous variables as part of maximum likelihood estimation or through
+        the Kalman filter (i.e. recursive least squares). If
+        `time_varying_regression` is True, this must be set to False.
+    simple_differencing : bool, optional, default=False
+        Whether or not to use partially conditional maximum likelihood
+        estimation. If True, differencing is performed prior to estimation,
+        which discards the first :math:`s D + d` initial rows but results in a
+        smaller state-space formulation. See the Notes section for important
+        details about interpreting results when this option is used. If False,
+        the full SARIMAX model is put in state-space form so that all
+        datapoints can be used in estimation.
+    enforce_stationarity : bool, optional, default=True
+        Whether or not to transform the AR parameters to enforce stationarity
+        in the autoregressive component of the model.
+    enforce_invertibility : bool, optional, default=True
+        Whether or not to transform the MA parameters to enforce invertibility
+        in the moving average component of the model.
+    hamilton_representation : bool, optional, default=False
+        Whether or not to use the Hamilton representation of an ARMA process
+        (if True) or the Harvey representation (if False).
+    concentrate_scale : bool, optional, default=False
+        Whether or not to concentrate the scale (variance of the error term)
+        out of the likelihood. This reduces the number of parameters estimated
+        by maximum likelihood by one, but standard errors will then not
+        be available for the scale parameter.
+    trend_offset : int, optional, default=1
+        The offset at which to start time trend values. Default is 1, so that
+        if `trend='t'` the trend is equal to 1, 2, ..., nobs. Typically is only
+        set when the model created by extending a previous dataset.
+    use_exact_diffuse : bool, optional, default=False
+        Whether or not to use exact diffuse initialization for non-stationary
+        states. Default is False (in which case approximate diffuse
+        initialization is used).
+
```

Comment:
pls also add a `See Also` section with references to `ARIMA`, `AutoARIMA` and `StatsForecastAutoARIMA`

Reply:
It's been added. I placed it above the References section